Issue24 chapter6 (#28)
metno/data-management-handbook: README.md

Data Management Handbook template for MET and partners in S-ENDA written in asciidoc.

See it [here](https://htmlpreview.github.io/?https://github.com/metno/data-management-handbook/blob/master/html/data-management-handbook.html).

This handbook template is based on the first version of the Data Management Handbook used at the Meteorological Institute in Norway (METNorway). The handbook was rewritten into this template version so that other institutions and partners could use it to create their own data management handbook.

This repo includes only the general parts of the data management handbook that should be universal for all partners, and templates for the chapters that include partner specific information. 

> 

Reviewer: ```suggestion

In case of questions and need for help - please submit an issue on this repository!

```

## How to use this handbook:

>  * Create a separate repo in Github or Gitlab, (If the data management handbook will include detailed information about IT systems, secrets or other information that should not be available outside of your organization, Gitlab is recommended)

Reviewer: ```suggestion
 * Create a separate repo in Github or Gitlab, (If the data management handbook will include detailed information about IT systems, secrets or other information that should not be available outside of your organization, a private repository is recommended)
```

>  * doc folder: Add only the templates for the chapters and appendices that includes partner specific information (all starting with specialized):

Reviewer: ```suggestion
   
   * doc folder: Add only the templates for the chapters and appendices that includes partner specific information (all starting with specialized):
   
```

	* specialized_intro_template.adoc
	* specialized_data_services_template.adoc
	* specialized_datagov_template.adoc
	* specialized_strudoc_template.adoc
	* specialized_userportals_template.adoc
	* specialized_practical_guidance_template.adoc
	* specialized_appendixB_template.adoc
	* To get your organisations visual profile on the DMH, create or add a .yml file with the theme for the handbook.

The repo also needs the following files:
 * gitlab-ci.yml
 * compile-docs.sh

The file “data-management-handbook.adoc”, includes the specialized files via variables. These variables are defined in the docker-compose.asciidoctor.yml file. (If one of the partner-specific files does not exist in your local repository, The CI will add the template). This is also the file that sets up the pdf.

## Compiling the documentation

> 

Reviewer: ```suggestion
The general part of the documentation can be compiled by running `vagrant up`. You can reuse the code here to add CI and compile your own version. Please ask if you need help.
```

> compile-docs.sh compiles the documentation. The document is compiled everytime something is committed.

Reviewer: ```suggestion
### Compilinig your own version (in the new repository)

- compile-docs.sh compiles the documentation.
```

> gitlab-ci.yml connects the files in your local repo to the environment variables. currently this is hardcoded, will perhaps be changed in the future.

Reviewer: ```suggestion
- gitlab-ci.yml connects the files in your local repo to the environment variables
```
